_posts/2023-06-05-opensearch-hadoop-launch.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Announcing the general availability of the OpenSearch Hadoop client"
+authors:
+  - hvamsi
+  - nknize
+date: 2023-05-06 12:15:00 -0700
+categories:
+  - releases
+meta_keywords: opensearch hadoop, apache spark, apache hive, apache hadoop, openseearch, mapreduce, hdfs
+meta_description: OpenSearch Hadoop is now generally available with support for multiple versions of OpenSearch to run on Spark and Hive.
+twittercard:
+  description: OpenSearch Hadoop is now generally available with support for multiple versions of OpenSearch to run on Spark and Hive.
+excerpt: We are excited to announce the release of the new OpenSearch Hadoop connector. This tool enables efficient interaction between your Hadoop-based Big Data operations and OpenSearch clusters, supporting all versions of OpenSearch.
+---
+
+We are excited to announce the release of the new OpenSearch-Hadoop connector. This tool enables efficient interaction between your Hadoop-based Big Data operations and OpenSearch clusters, supporting all versions of OpenSearch.
+
+## OpenSearch Hadoop connector features:
```

Comment:
Can we add an opening paragraph here? For example, "The OpenSearch-Hadoop connector includes the following features:" (and remove the colon from the heading)

Reply:
> This is a good start. I like having the compatibility matrices. Might be good, though, to also add a simple "Getting Started Examples"?
> 
> Maybe an example on how to write to a dataframe in scala?
> 
> e.g.,
> 
> ```
> val spark = SparkSession.builder().master("local[*]")
>     .config("opensearch.nodes", "127.0.0.1").config("opensearch.net.http.auth.user", "admin").config("opensearch.net.http.auth.pass", "admin").config("opensearch.net.ssl", "true")
>     .config("opensearch.batch.size.bytes", "1kb").config("opensearch.net.ssl.cert.allow.self.signed", "true")
>     .getOrCreate()
> ```
> 
> or how to use it with pyspark like I demonstrate in my [comment on #153](https://github.com/opensearch-project/opensearch-hadoop/issues/153#issuecomment-1551501905).
> 
> I'm happy to add if you'd like.

Go for it!